about/distribution/index.markdown

```diff
@@ -0,0 +1,72 @@
+    <h2>List of Distributions</h2>
+    <table class="table table-striped table--distribution">
+      <thead>
+        <tr>
+          <th>Distro</th>
+          <th>Release date</th>
+          <th>Logo</th>
+          <th>EOL date</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td><a href="https://moveit.picknik.ai/main/index.html">MoveIt Rolling</a></td>
+          <td>2021</td>
+          <td><img src="/assets/images/distribution/moveIt-rolling.jpg" alt="MoveIt Rolling" height="100" /></td>
+          <td>N/A</td>
+        </tr>
+        <tr>
+          <td><a href="https://moveit.picknik.ai/galactic/index.html">MoveIt 2 Galactic</a></td>
+          <td>July 8th, 2021</td>
+          <td><img src="/assets/images/distribution/galactic.png" alt="MoveIt 2 Galactic" height="100" /></td>
+          <td>November 2022</td>
+        </tr>
+        <tr>
+          <td><a href="https://moveit.picknik.ai/foxy/index.html">MoveIt 2 Foxy Fitzroy</a></td>
+          <td>June 5th, 2020</td>
+          <td><img src="/assets/images/distribution/moveit2-foxy.png" alt="MoveIt 2 Foxy Fitzroyc" height="100" /></td>
+          <td>May 2023</td>
+        </tr>
+        <tr>
+          <td><a href="https://ros-planning.github.io/moveit_tutorials/">MoveIt 1 Noetic</a></td>
+          <td>October 13th, 2020</td>
+          <td><img src="/assets/images/distribution/MoveIt.png" alt="MoveIt 1 Noetic" height="100" /></td>
+          <td>May 2023</td>
+        </tr>
+        <tr>
+          <td><a href="http://docs.ros.org/en/melodic/api/moveit_tutorials/html/index.html">MoveIt 1 Melodic</a></td>
+          <td>June 5th, 2020</td>
+          <td><img src="/assets/images/distribution/MoveIt.png" alt="MoveIt 1 Melodic" height="100" /></td>
+          <td>May 2023</td>
+        </tr>
+      </tbody>
+    </table>
+    <h2>Distribution Details</h2>
+    <p>For details on the distributions see each release page. For the supported platforms and versions of common dependencies and other considerations, see the official ROS 2 Target Platforms REP 2000.</p>
+    <h2>Future Distributions</h2>
+    <p>For details on upcoming features see the roadmap.</p>
+    <p>Currently there is a new ROS 2 distribution roughly every year. The following information is a best estimate and is subject to change.</p>
+    <p>The expectation is to release new ROS 2 distributions once per year.</p>
+    <h2>Rolling Distribution</h2>
+    <p>The Rolling distribution of ROS 2 serves as a staging area for future stable distributions of ROS 2 and as a collection of the most recent development releases. Unlike most stable ROS 2 distributions which have an initial release, a support window during which they are updated, and a definite end of support (see List of Distributions above) the Rolling distribution is continuously updated and is subject to in-place updates which will at times include breaking changes.</p>
```

Comment:
This is more or less copied from https://docs.ros.org/en/rolling/Releases.html. afaik, google really doesn't like redundant content (verbatim) and might downrank the page because of this, especially if we reference the ROS 2 distro page directly.